uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/ZonesProperties.java

```diff
@@ -0,0 +1,21 @@
+package org.cloudfoundry.identity.uaa.orchestrator;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Data
+@Lazy
+@Component
+@ConfigurationProperties(prefix = "zones")
+public class ZonesProperties {
```

Comment:
why you need this class? i dont see any zones properties configs in uaa/src/test/resources/orchestrator-integration.yml

Reply:
Done